samples/plugins/mysql_odbc/manifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='mysql_odbc_connector' superclass='mysql_odbc' plugin-version='0.0.0' name='MySQL ODBC Plugin Connector' version='18.1'>
+  <vendor-information>
+    <company name='TestCompany' />
+    <support-link url='http://google.com' />
```

Comment:
http://example.com/  It's what RFCs use for sample urls.  It resolves but it also clearly a sample.